src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -227,13 +231,20 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
             // create one here
             b.setField(fields);
             bibitems.add(b);
-
         }
 
         return new ParserResult(bibitems);
 
     }
 
+    private boolean checkLineValidity(String line) {
+        if (line.length() >= 4) {
+            return line.contains("-") || !(line.charAt(4) == '-') ? true : false;
```

Comment:
Also please check the if statement again, I can't find a case where this will return `false`.